uat/steps/test_e2e.py

```diff
@@ -10,14 +10,14 @@ def verify_test_files(context):
     assert Path(cwd).joinpath(GDK_TEST_DIR).resolve().exists(), f"{GDK_TEST_DIR} does not exist"
 
 
-@step("we verify that the OTF version used is {version}")
+@step("we verify that the GTF version used is {version}")
 def verify_test_framework_version(context, version):
     cwd = context.cwd if "cwd" in context else os.getcwd()
     build_system_file = Path(cwd).joinpath(GDK_TEST_DIR, "pom.xml")
 
     with open(build_system_file, "r") as f:
         content = f.read()
-        assert f"<otf.version>{version}-SNAPSHOT</otf.version>" in content, f"OTF version {version} is not used."
```

Comment:
is it meant to still use the `otf.version` parameter?

Reply:
Yup. That's actually still in the template, so it has to stay like that until the template gets updated.